src/components/MapView/MapView.web.tsx

```diff
@@ -77,40 +148,55 @@ const MapView = forwardRef<MapViewHandle, MapViewProps>(
         );
 
         return (
-            <View
-                style={style}
-                // eslint-disable-next-line
-                {...responder.panHandlers}
-            >
-                <Map
-                    ref={setRef}
-                    mapLib={mapboxgl}
-                    mapboxAccessToken={accessToken}
-                    initialViewState={{
-                        longitude: initialState.location[0],
-                        latitude: initialState.location[1],
-                        zoom: initialState.zoom,
-                    }}
-                    style={StyleUtils.getTextColorStyle(themeColors.mapAttributionText) as React.CSSProperties}
-                    mapStyle={styleURL}
-                >
-                    {waypoints?.map(({coordinate, markerComponent, id}) => {
-                        const MarkerComponent = markerComponent;
-                        return (
-                            <Marker
-                                key={id}
-                                longitude={coordinate[0]}
-                                latitude={coordinate[1]}
-                            >
-                                <MarkerComponent />
-                            </Marker>
-                        );
-                    })}
-                    {directionCoordinates && <Direction coordinates={directionCoordinates} />}
-                </Map>
-            </View>
+            <>
+                {!isOffline && Boolean(accessToken) && Boolean(currentPosition) ? (
+                    <View
+                        style={style}
+                        // eslint-disable-next-line react/jsx-props-no-spreading
+                        {...responder.panHandlers}
+                    >
+                        <Map
+                            onDrag={() => setUserInteractedWithMap(true)}
```

Comment:
Are there any other options for user to interact with the map?

Reply:
I only care about user dragging the map - if he drags then I want to stop panning the map. If he clicks/presses/zooms or anything else, we are fine with panning the map. This made me think - maybe a better name is `setUserInteractedWithMap` -> `setUserHasDraggedMap`

Reply:
Yeah, sounds better this way